### ISSUE TITLE
Slather includes test classes to the coverage report

### ISSUE THREAD

**Issue by poborin** [SlatherOrg/slather#307]
slather 2.4.1

By executing the command
```
slather coverage --simple-output --scheme MyFramework MyFramework.xcodeproj
```
I'm getting the following report:

```
Slathering...
MyFramework/AboutPage.swift: 29 of 31 lines (93.55%)
MyFrameworkTests/AboutPageTests.swift: 60 of 60 lines (100.00%)
Test Coverage: 96.78%
Slathered
```

I asume test classes should be excluded by default. Thus the final coverage will be less

**Comment by ksuther** [comment]
Test cases are included in the llvm-cov output. If you want slather to ignore those files use `-i` to ignore them.

**Comment by poborin** [comment]
hmm... I don't remember test classes included in previous versions.

**Comment by ksuther** [comment]
The slather test cases include the test classes. They've definitely been there all along.

**Comment by dbrajkovic** [comment]
They should include the test classes.   If the code in a test class isn't getting called, then why is it there?

**Comment by NayaraAlves2912** [comment]
When I run the slather only appears the test classes, can someone help me?

`slather coverage --simple-output --scheme project --workspace path/workspace.wcworkspace path/project.xcodeproj`

**Comment by mikelupo** [comment]
Did you try —ignore? 
Incase my email munched the dashes, it’s minus minus ignore <ant style path>

**Comment by NayaraAlves2912** [comment]
Yes, but only the test classes appeared, to solve this it was necessary to enter the source of the project, so I was able to capture both classes and use the ignore for test classes

**Comment by lexjhuang** [comment]
Same issue I encounter, it is very tricky when I am using a static framework in a workspace.
But I finally got the report by doing below code.

xcodebuild test -UseModernBuildSystem=NO -configuration Debug -scheme SubScheme -destination 'platform=iOS Simulator,name=iPhone SE,OS=12.1'
slather coverage --html --scheme MainScheme --workspace MainScheme .xcworkspace MainScheme /MainScheme .xcodeproj

The point is that when you are doing the test, you need to execute the sub module test target, but when you are getting the coverage, you need to specify the main scheme instead

reference issue [https://github.com/SlatherOrg/slather/issues/408](url)
